labgrid/driver/usbvideodriver.py

```diff
@@ -54,7 +63,7 @@ def stream(self, caps_hint=None):
         tx_cmd = self.video.command_prefix + ["gst-launch-1.0", "-q"]
         tx_cmd += pipeline.format(self.video.path, caps).split()
         rx_cmd = ["gst-launch-1.0"]
-        rx_cmd += "fdsrc ! h264parse ! avdec_h264 ! glimagesink sync=false".split()
+        rx_cmd += "playbin uri=fd://0 sync=false".split()
```

Comment:
`playbin` has no `sync` property. And it should not be necessary here anyways.

Reply:
OK. I've removed the sync=false.